supplementary_style_guide/style_guidelines/lead-in-sentences.adoc

```diff
@@ -0,0 +1,11 @@
+[[lead-in-sentences]]
+= Lead-in sentences
+
+A lead-in sentenece is the text that directly follows a procedure or prerequiste heading in a task-based module. It is not the module abstract, which describes the goals of the user for the module.
+
+.Example lead-in sentences
```

Comment:
Singular, since it's just one. And maybe specify that it's for a procedure? So ".Example lead-in sentence for the `Procedure` section" ?

Or maybe someone else has a better way of phrasing this all to be clearer about which lead-in sentences we mean...

Reply:
```suggestion
A lead-in sentence immediately precedes the first step in a procedure and is in the form of "To do something...:".
```

I don't recall lead-in sentences being part of a prerequisite.

Reply:
@joaedwar  If they are part of a prereq, please provide an example specifically for this case.